libs/tdigest/src/test/java/org/elasticsearch/tdigest/SortingDigestTests.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ * This project is based on a modification of https://github.com/tdunning/t-digest which is licensed under the Apache 2.0 License.
+ */
+
+package org.elasticsearch.tdigest;
+
+import org.junit.BeforeClass;
+
+public class SortingDigestTests extends TDigestTests {
```

Comment:
This is the slowest test suite on my machine. Looks like `testMonotonicity(...)` needed 28 seconds.

Reply:
You need a new laptop :)

Changed the granularity to run 10x faster. It takes 0.25ms on my laptop, I can make it even faster as needed.

Reply:
I will inform the IT department :)